lib/sources/subscription_watch_info.rb

```diff
@@ -0,0 +1,26 @@
+module Sources
+  class SubscriptionWatchInfo
+    XRHID = Base64.strict_encode64({:identity => {:account_number => "12345", :user => {:is_org_admin => true}}}.to_json).freeze
```

Comment:
`account_number` should be hardcoded ?

Reply:
Yeah this is just a dummy header to fetch from their Internal API, the acct # doesn't matter.  

Might move it out to a svc class haven't decided yet.

Reply:
👍 thanks